src/averageAges.js

```diff
@@ -38,6 +55,16 @@ function calculateMenAverageAge(people, century) {
  */
 function calculateWomenAverageAge(people, withChildren) {
   // write code here
+  const women = people.filter(person =>
+    person.sex === 'f'
+    && (withChildren
+      ? people.find(child => child.mother === person.name)
+      : true
+    ));
+
+  const wAges = women.map(woman => woman.died - woman.born);
```

Comment:
is that what's stored inside the variable?

![image](https://github.com/mate-academy/js_average-ages/assets/103033758/0be2ed81-0ef1-4800-9a0f-ec486e0c05a3)